Script/PRONTO.py

```diff
@@ -942,6 +1132,8 @@ def main(argv):
 	DNA_normal_sampleID = ""
 	remisse_mail = False
 	update_clinical_file = False
+	target_cod_region = 0
```

Comment:
did you mean?
```suggestion
	target_code_region = 0
```

Reply:
I might have missed it, but is this actually used somewhere?

Reply:
Ah... Some typing error I will fix later. :)